src/frontend/next/src/components/SEO.tsx

```diff
@@ -0,0 +1,47 @@
+import { Helmet } from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+type SEOProps = {
+  description?: string;
+  lang?: string;
+  meta?: Array<any>;
+  title?: string;
+};
+const SEO = ({ description, lang, meta, title }: SEOProps) => {
+  const siteMetadata = useSiteMetadata();
+  const metaDescription = description || siteMetadata.description;
+  const concatMeta = [
+    {
+      name: `description`,
+      content: metaDescription,
+    },
+    {
+      property: `og:title`,
+      content: title,
+    },
+    {
+      property: `og:description`,
+      content: metaDescription,
+    },
+  ].concat(meta ?? []);
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang,
+      }}
+      title={title}
+      titleTemplate={`%s | ${siteMetadata.title}`}
+      meta={concatMeta}
+    />
+  );
```

Comment:
I think you should use 'next/head' for SEO instead of Helmet. 
Look at the very bottom of this page: https://nextjs.org/docs/migrating/from-gatsby

Reply:
@foxviet : Thanks for the review. This has been discussed in previous comments and I will update it to Next style in next commit